src/Shared/SemanticConventions.cs

```diff
@@ -7,116 +7,23 @@ namespace OpenTelemetry.Trace;
-    public const string AttributeDbJdbcDriverClassName = "db.jdbc.driver_classname";
-    public const string AttributeDbName = "db.name";
-    public const string AttributeDbStatement = "db.statement";
-    public const string AttributeDbOperation = "db.operation";
     public const string AttributeDbInstance = "db.instance";
-    public const string AttributeDbUrl = "db.url";
-    public const string AttributeDbCassandraKeyspace = "db.cassandra.keyspace";
-    public const string AttributeDbHBaseNamespace = "db.hbase.namespace";
-    public const string AttributeDbRedisDatabaseIndex = "db.redis.database_index";
-    public const string AttributeDbMongoDbCollection = "db.mongodb.collection";
-
-    public const string AttributeRpcSystem = "rpc.system";
-    public const string AttributeRpcService = "rpc.service";
-    public const string AttributeRpcMethod = "rpc.method";
-    public const string AttributeRpcGrpcStatusCode = "rpc.grpc.status_code";
 
     public const string AttributeMessageType = "message.type";
     public const string AttributeMessageId = "message.id";
-    public const string AttributeMessageCompressedSize = "message.compressed_size";
-    public const string AttributeMessageUncompressedSize = "message.uncompressed_size";
-
-    public const string AttributeFaasTrigger = "faas.trigger";
-    public const string AttributeFaasExecution = "faas.execution";
-    public const string AttributeFaasDocumentCollection = "faas.document.collection";
-    public const string AttributeFaasDocumentOperation = "faas.document.operation";
-    public const string AttributeFaasDocumentTime = "faas.document.time";
-    public const string AttributeFaasDocumentName = "faas.document.name";
-    public const string AttributeFaasTime = "faas.time";
-    public const string AttributeFaasCron = "faas.cron";
-
-    public const string AttributeMessagingSystem = "messaging.system";
-    public const string AttributeMessagingDestination = "messaging.destination";
-    public const string AttributeMessagingDestinationKind = "messaging.destination_kind";
-    public const string AttributeMessagingTempDestination = "messaging.temp_destination";
-    public const string AttributeMessagingProtocol = "messaging.protocol";
-    public const string AttributeMessagingProtocolVersion = "messaging.protocol_version";
-    public const string AttributeMessagingUrl = "messaging.url";
-    public const string AttributeMessagingMessageId = "messaging.message_id";
-    public const string AttributeMessagingConversationId = "messaging.conversation_id";
-    public const string AttributeMessagingPayloadSize = "messaging.message_payload_size_bytes";
-    public const string AttributeMessagingPayloadCompressedSize = "messaging.message_payload_compressed_size_bytes";
-    public const string AttributeMessagingOperation = "messaging.operation";
 
     public const string AttributeExceptionEventName = "exception";
```

Comment:
I guess these are probably the only thing we need, maybe just put these in the actual implementation (not shared) rather than keeping a shared file.

Reply:
Part of the attributes is used a least in 2 packages. I have tried just to remove unused things (at least in this PR).